SU2_CFD/include/numerics_structure.hpp

```diff
@@ -5521,4 +5521,109 @@ class CViscous_Template : public CNumerics {
   void ComputeResidual(su2double *val_residual, su2double **val_Jacobian_i, su2double **val_Jacobian_j, CConfig *config);
 };
 
+/*!
+ * \class CDiscGalerkin_Flow
```

Comment:
I put these numerics classes in the original skeleton when we first started with the DG work, but I don't think we ever needed them, since we wanted to reduce the number of levels of indirection. In the end, there are no numerics or variable classes, with one exception for the existing Riemann solvers in numerics. Otherwise, everything is in the solver class to improve performance (a lesson we learned the first time around with Intel in the FVM solver).

Can you please remove these classes from the header and the numerics_direct_mean_fem.cpp file if they are no longer needed? Any other artifacts of CVariable or CNumerics for the DG solver should be removed if possible, apart from the needed Riemann solvers.